tests/core_test.py

```diff
@@ -32,6 +32,7 @@
 from jax.core import UnshapedArray, ShapedArray
 from jax.tree_util import (tree_flatten, tree_unflatten, tree_map, tree_reduce,
                            tree_leaves)
+from jax.api_util import flatten_fun_nokwargs
```

Comment:
Can remove

Reply:
Hm it's used in the commented-out code... maybe instead of commenting it out I'll just leave it as an unused function.